### ISSUE TITLE
About the MOCKINGJAY model

### ISSUE THREAD

**Issue by whmrtm** [andi611/Self-Supervised-Speech-Pretraining-and-Representation-Learning#50]
Hi Andy,
Thanks for all the amazing works!
I just have a question about the Mockingjay model. It is mentioned in the paper that a linear-scale spectrogram is also used as the output reconstruction target. Do you have any results showing the difference between linear spectrogram and melspectrogram? Also, is it possible that you provide some pre-trained models for the linear spectrogram? I think it would be more convenient to some more complex downstream tasks like deep perceptual loss.

Best Regards

**Comment by andi611** [comment]
Hi,
> Do you have any results showing the difference between linear spectrogram and melspectrogram?

Empirically, I find reconstruction target has little effect on phone classification or speaker classification.

> Also, is it possible that you provide some pre-trained models for the linear spectrogram? 

I am currently training a new model with linear as output reconstruction target.
I'm using the following setting:
```
win_ms: 25
hop_ms: 10
n_freq: 201
n_mels: 80

input:
  feat_type: mel
  log: True
  delta: 1
  cmvn: True

target:
  feat_type: linear
  log: True
  delta: 0
  cmvn: True
```

Hope you will find it useful. I'll comment here once its done.

**Comment by andi611** [comment]
The model is now available in the Mockingjay directory under the S3PRL Google Drive.
The name of the ckpt is: `on-the-fly-melLinearBase960-b12-d03-T-libri`
A simply tutorial on how to use this model can be found [here](https://github.com/andi611/Self-Supervised-Speech-Pretraining-and-Representation-Learning/blob/master/src/tutorial_use_pretrained_model_without_preprocessing.py).
